kratos/tests/cpp_tests/utilities/test_debug_helpers.cpp

```diff
@@ -0,0 +1,35 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+//
+
+// Project includes
+#include "includes/debug_helpers.h"
+#include "includes/ublas_interface.h"
+#include "testing/testing.h"
+
+namespace Kratos
+{
+namespace Testing
+{
+KRATOS_TEST_CASE_IN_SUITE(DebugHelpersKRATOS_WATCH_VECTOR_WITH_PRECISION, KratosCoreFastSuite)
+{
+    Vector temp(4, 10.0);
+    KRATOS_WATCH_VECTOR_WITH_PRECISION(temp, 12);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(DebugHelpersKRATOS_WATCH_MATRIX_WITH_PRECISION, KratosCoreFastSuite)
+{
+    Matrix temp(4, 4, 10.0);
+    KRATOS_WATCH_MATRIX_WITH_PRECISION(temp, 12);
```

Comment:
this doesn't actually test sth so I think it can be removed

Reply:
I put it there because, debug_helpers.h is headers only, therefore it is not even compiled if it is not included. So if someone does changes, this test will check whther the defines are atleast working. I dont know how to check whether the ouput is as it suppose to be.

Example output looks like this:

TestDebugHelpersKRATOS_WATCH_MATRIX_WITH_PRECISIONtemp : 
temp(0, 0) = 1.000000000000e+01;
temp(0, 1) = 1.000000000000e+01;
temp(0, 2) = 1.000000000000e+01;
temp(0, 3) = 1.000000000000e+01;
temp(1, 0) = 1.000000000000e+01;
temp(1, 1) = 1.000000000000e+01;
temp(1, 2) = 1.000000000000e+01;
temp(1, 3) = 1.000000000000e+01;
temp(2, 0) = 1.000000000000e+01;
temp(2, 1) = 1.000000000000e+01;
temp(2, 2) = 1.000000000000e+01;
temp(2, 3) = 1.000000000000e+01;
temp(3, 0) = 1.000000000000e+01;
temp(3, 1) = 1.000000000000e+01;
temp(3, 2) = 1.000000000000e+01;
temp(3, 3) = 1.000000000000e+01;

                      OK.

TestDebugHelpersKRATOS_WATCH_VECTOR_WITH_PRECISIONtemp : 
temp[0] = 1.000000000000e+01;
temp[1] = 1.000000000000e+01;
temp[2] = 1.000000000000e+01;
temp[3] = 1.000000000000e+01;

                      OK.